operate/client/src/App/ProcessInstance/v2/index.tsx

```diff
@@ -150,43 +156,38 @@ const ProcessInstance: React.FC = observer(() => {
 
   useEffect(() => {
     let processTitleDisposer = when(
-      () => processInstanceDetailsStore.processTitle !== null,
+      () => !!processTitle,
       () => {
-        document.title = processInstanceDetailsStore.processTitle ?? '';
+        document.title = processTitle ?? '';
       },
     );
 
     return () => {
       processTitleDisposer();
     };
-  }, []);
-
-  const {
-    state: {processInstance},
-  } = processInstanceDetailsStore;
+  });
 
   const {
     isModificationModeEnabled,
     state: {modifications, status: modificationStatus},
   } = modificationsStore;
 
-  const isBreadcrumbVisible =
-    processInstanceDetailsStore.state.processInstance !== null &&
-    processInstanceDetailsStore.state.processInstance?.callHierarchy?.length >
-      0;
+  const isBreadcrumbVisible = callHierarchy && callHierarchy.items.length > 0;
 
   const hasPendingModifications = modifications.length > 0;
 
   const {
     state: {isListenerTabSelected},
   } = processInstanceListenersStore;
 
-  if (processInstanceDetailsStore.state.status === 'forbidden') {
+  if (error?.response?.status === 403) {
```

Comment:
I'm wondering if it isn't better to have a const with it even though this value is standard/universal? :thinking: 

```suggestion
  if (error?.response?.status === FORBIDDEN_STATUS_CODE) {
```

Reply:
That's a good point. Even if it's universal, it could make the code more readable and give semantic meaning to it. I've defined a new const and updated this occurrence. There are other occurrences across the code (of 403, 404 etc...) which I think we could migrate in a follow-up PR.